site/src/js/index.js

```diff
@@ -51,7 +51,8 @@ function handleTags() {
     }
 
     Array.from(contentEl.querySelectorAll('.item')).forEach(el => {
-        const show = tags.reduce((sho, tag) => sho || el.classList.contains(`tag-${tag}`), false);
+        const show = tags.reduce((sho, tag) => sho || el.classList.contains(`tag-${tag}`), false) && !(![...tags].includes("tag-expired") && el.classList.contains("tag-expired"));
```

Comment:
I don't think copying the array is necessary. 

```suggestion
        const show = tags.reduce((sho, tag) => sho || el.classList.contains(`tag-${tag}`), false) && !(!tags.includes("tag-expired") && el.classList.contains("tag-expired"));
```